autoload/LanguageClient.vim

```diff
@@ -1401,12 +1418,10 @@ endfunction
 
 function! LanguageClient_contextMenu() abort
     let l:options = keys(LanguageClient_contextMenuItems())
-
-    if get(g:, 'loaded_fzf') && get(g:, 'LanguageClient_fzfContextMenu', 1)
-        return fzf#run(fzf#wrap({
-                    \ 'source': l:options,
-                    \ 'sink': function('LanguageClient_handleContextMenuItem'),
-                    \ }))
+    if type(get(g:, 'LanguageClient_selectionUI', v:null)) is s:TYPE.funcref
+                \ || get(g:, 'LanguageClient_selectionUI', v:null) ==? 'FZF'
+                \ || get(g:, 'loaded_fzf')
```

Comment:
Same here, this will always fall in the if branch if fzf is loaded. I would do something like this.

```vim
if type(get(g:, 'LanguageClient_selectionUI', v:null)) is s:TYPE.funcref
    \ || (get(g:, 'LanguageClient_selectionUI', v:null) ==? 'FZF' && get(g:, 'loaded_fzf'))
```

Reply:
I just realized I completely dropped the `LanguageClient_fzfContextMenu` check. :flushed: So the final check would be:

```vim
if get(g:, 'LanguageClient_fzfContextMenu', 1)
    \ && (type(get(g:, 'LanguageClient_selectionUI', v:null)) is s:TYPE.funcref
        \ || (get(g:, 'LanguageClient_selectionUI', v:null) ==? 'FZF' && get(g:, 'loaded_fzf')))
```

This is just your version with the `LanguageClient_fzfContextMenu` check in front.
 I'm gonna add that back in, too, and alias it to `LanguageClient_selectionUIContextMenu` or something in a separate commit (naming suggestions welcome, of course).